decidim/decidim
# Fix missing fields on duplicate meetings functionality

#### :tophat: What? Why?

Fix the copy meeting functionality in 0.28.

#### :pushpin: Related Issues

- Fixes #9848

#### Testing

1. Go to 'Meetings components (admin)'
2. Click on 'Duplicate meeting'
3. Change the date and some other information, check that meeting type is set to hybrid, scroll down to click Copy button
4. The meeting is created as In person type
 
:hearts: Thank you!
